### ISSUE TITLE
For some reason, it stopped using python3

### ISSUE THREAD

**Issue by rschwiebert** [spotify/dh-virtualenv#282]
I've been using the same debian package contents to successfully build with python3 for months, but for some reason its behavior changed.

I'm using ansible and vagrant to provision Ubuntu 16.04 vms.

On one working vm, I get this output at the beginning:

```make[1]: Entering directory '/opt/vagrant/blue-server/mything'
dh_virtualenv -p python-mything
Using real prefix '/usr'
Path not in prefix '/opt/venvs/py3/include/python3.5m' '/usr'
New python executable in /opt/vagrant/blue-server/mything/debian/python-mything/opt/venvs/python-mything/bin/python3
Also creating executable in /opt/vagrant/blue-server/mything/debian/python-mything/opt/venvs/python-mything/bin/python
```
and the package builds successfully.

On the one that doesn't behave, I get
```make[1]: Entering directory '/opt/vagrant/blue-server/mything'
dh_virtualenv -p python-mything
Running virtualenv with interpreter /usr/bin/python2
New python executable in /opt/vagrant/blue-server/mything/debian/python-mything/opt/venvs/python-mything/bin/python2
Also creating executable in /opt/vagrant/blue-server/mything/debian/python-mything/opt/venvs/python-mything/bin/python
```
and the build fails later on.

Since then I have reread the docs on building a python3 virtualenv with dh-virtualenv and made sure everything was followed accurately, but nothing changes. Rebuilding the vm also doesn't help.

Any idea what I can do to debug this?  I really have no idea what the difference between the two vm's is. My working theory now is that something changed in the ubuntu repositories that borked things and made a difference when I provisioned.

The code that dh_virtualenv is operating on is definitely identical in both cases, so it has to be the environment somehow.

I was able to force it back to using python3 by including `--python /usr/bin/python3`, but it would still be nice to know why it's not working as it used to.

Another possible clue... in the one that doesn't work, it appears to be based on pip==8, and therefore can't parse the pip.conf, and it breaks.  On the other vm, whatever pip its using apparently is no problem, because it parses pip.conf without a problem.

**Comment by rschwiebert** [comment]
Ugh, I think I finally stumbled upon the answer!

In the working environment, virtualenv==16.5.0 is available (the previously available version was 15.0.1 in the system packages), and adding it to the environment of the second one seems to get things to work.

So, in summary, the problem is that I could not use a pip.conf without virtualenv==16.5.0 in the system packages.  I also confirmed that manually specifying all the extra-index-urls (and blowing away the pip.conf) also solved the problem.
